PrintingPodRecharge/Mod.cs

```diff
@@ -81,7 +81,7 @@ private void CreateConfigDirectory()
         public override void OnAllModsLoaded(Harmony harmony, IReadOnlyList<KMod.Mod> mods)
         {
             base.OnAllModsLoaded(harmony, mods);
-            DataGen.BundleGen.Generate(Path.Combine(ModAssets.GetRootPath(), "data", "bundles"), true);
+            DataGen.BundleGen.Generate(Path.Combine(ModAssets.GetRootPath(), "data", "bundles"), false);
```

Comment:
This is the actual fix

Reply:
If you discovered a bug with the functionality please open an issue so i can look into it. 
The point of putting them outside is to let users modify the packs. By forcing this, you are overwriting the user made jsons, defeating the point entirely. 
If I always wanted to force pack data, i would put them in my mod folder, which gets force updated anyway with each mod updater.

Reply:
The actual bug is that in the **current**, live version of the code it is forcing a write overwriting the data, which I fixed in this pull request by changing it to **not** overwrite it at every start. The last function parameter is called `force` and if it is ture then the files will get overwritten even if they exist, see here: https://github.com/aki-art/ONI-Mods/blob/bio-inks/PrintingPodRecharge/DataGen/BundleGen.cs#L35-L58